client/app/billing/payment/method/add/index.js

```diff
@@ -0,0 +1,33 @@
+import angular from 'angular';
+
+import 'angular-translate';
+import 'ovh-ui-angular';
+import '@ovh-ux/ng-translate-async-loader';
+import '@ovh-ux/ng-ovh-payment-method';
+import '@ovh-ux/manager-core';
+
+import routing from './routing';
+import component from './component';
+
+import billingContactViewModule from './views/billingContact';
+import legacyBankAccountViewModule from './views/legacyBankAccount';
```

Comment:
Sound weird to have `modules` in `/views` 😄

Reply:
I wasn't sure about how to do because I didn't know how to handle the translations of specific views.
During run of this module maybe?

Reply:
I meant the folder name is ambiguous, and translations should be specific to module and you should be able to run your module without its parent